### PR TITLE
ZH-trans: add 2017-11-00-Autoscaling-In-Kubernetes.md

### PR DESCRIPTION
**Pull request by xichengliudui**
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> Please delete this note before submitting the pull request.
>
> For 1.14 Features: set Milestone to 1.14 and Base Branch to dev-1.14
> 
> For Chinese localization, base branch to release-1.12
>
> For Korean Localization: set Base Branch to dev-1.13-ko.<latest team milestone>
>
> Help editing and submitting pull requests:
> https://kubernetes.io/docs/contribute/start/#improve-existing-content.
>
> Help choosing which branch to use:
> https://kubernetes.io/docs/contribute/start#choose-which-git-branch-to-use.
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>

### CODE REVIEW

File: content/zh/blog/_posts/2017-11-00-Autoscaling-In-Kubernetes.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+<!--
+Enjoyed this talk? Join us for more exciting sessions on scaling and automating your Kubernetes clusters at KubeCon in Austin on December 6-8. [Register Now](https://www.eventbrite.com/e/kubecon-cloudnativecon-north-america-registration-37824050754?_ga=2.9666039.317115486.1510003873-1623727562.1496428006)  
+-->
+喜欢这个演讲吗？ 12月6日至8日，在 Austin 参加 KubeCon 关于扩展和自动化您的 Kubernetes 集群的更令人兴奋的会议。[现在注册](https://www.eventbrite.com/e/kubecon-cloudnativecon-north-america-registration-37824050754?_ga=2.9666039.317115486.1510003873-1623727562.1496428006)。
+
+<!--
+Be sure to check out [Automating and Testing Production Ready Kubernetes Clusters in the Public Cloud](http://sched.co/CU64) by Ron Lipke, Senior Developer, Platform as a Service, Gannet/USA Today Network.
+-->
+一定要查看由 Ron Lipke， Gannet/USA Today Network, 平台即服务高级开发人员，在[公共云中自动化和测试产品就绪的 Kubernetes 集群](http://sched.co/CU64)。
```
Comment (xichengliudui) [inline content/zh/blog/_posts/2017-11-00-Autoscaling-In-Kubernetes.md]: `Today Network` 今天的网络？

Comment (chenrui333) [reply content/zh/blog/_posts/2017-11-00-Autoscaling-In-Kubernetes.md]: USA Today Network
USA Today 是一个美国报刊，就跟China Today 一样。
你这里直接用英文原文吧。

File: content/zh/blog/_posts/2017-11-00-Autoscaling-In-Kubernetes.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+<!--
+Kubernetes allows developers to automatically adjust cluster sizes and the number of pod replicas based on current traffic and load. These adjustments reduce the amount of unused nodes, saving money and resources. In this talk, Marcin Wielgus of Google walks you through the current state of pod and node autoscaling in Kubernetes: .how it works, and how to use it, including best practices for deployments in production applications.  
+-->
+Kubernetes 允许开发人员根据当前的流量和负载自动调整集群大小和 pod 副本的数量。这些调整减少了未使用节点的数量，节省了资金和资源。
+在这次演讲中，谷歌的 Marcin Wielgus 将带领您了解 Kubernetes 中 pod 和 node 自动调焦的当前状态：它是如何工作的，以及如何使用它，包括在生产应用程序中部署的最佳实践。
+
+<!--
+Enjoyed this talk? Join us for more exciting sessions on scaling and automating your Kubernetes clusters at KubeCon in Austin on December 6-8. [Register Now](https://www.eventbrite.com/e/kubecon-cloudnativecon-north-america-registration-37824050754?_ga=2.9666039.317115486.1510003873-1623727562.1496428006)  
+-->
+喜欢这个演讲吗？ 12月6日至8日，在 Austin 参加 KubeCon 关于扩展和自动化您的 Kubernetes 集群的更令人兴奋的会议。[现在注册](https://www.eventbrite.com/e/kubecon-cloudnativecon-north-america-registration-37824050754?_ga=2.9666039.317115486.1510003873-1623727562.1496428006)。
```
Comment (chenrui333) [inline content/zh/blog/_posts/2017-11-00-Autoscaling-In-Kubernetes.md]:
```suggestion
喜欢这个演讲吗？ 12 月 6 日至 8 日，在 Austin 参加 KubeCon 关于扩展和自动化您的 Kubernetes 集群的更令人兴奋的会议。[现在注册](https://www.eventbrite.com/e/kubecon-cloudnativecon-north-america-registration-37824050754?_ga=2.9666039.317115486.1510003873-1623727562.1496428006)。
```

Comment (chenrui333) [reply content/zh/blog/_posts/2017-11-00-Autoscaling-In-Kubernetes.md]: 我都忘记了，这个数字之间是要space的吧？

Comment (xichengliudui) [reply content/zh/blog/_posts/2017-11-00-Autoscaling-In-Kubernetes.md]: 数字之前确实需要 space，但是这样看起来很奇怪，我修改一下。

File: content/zh/blog/_posts/2017-11-00-Autoscaling-In-Kubernetes.md (hunk 1 above)
Comment (xichengliudui) [reply content/zh/blog/_posts/2017-11-00-Autoscaling-In-Kubernetes.md]: 长见识。

**Comment by fleeto** [comment]
`[trans-bot:N/A]`

Thank you @xichengliudui, the [related task issue](https://github.com/k8smeetup/website-tasks/issues/893) has no assignee. will not be reported to the task issues

**Pull request merged**